src/foam/nanos/ruler/RuleHistory.js

```diff
@@ -42,6 +54,12 @@ foam.CLASS({
       visibility: 'RO',
       documentation: 'Id of the object on which rule is applied.'
     },
+    {
+      class: 'Long',
+      name: 'longObjectId',
+      visibility: 'RO',
+      documentation: 'Long type version of objectId'
```

Comment:
@nanoCarl .. can you add a more detailed description of why this is here in the documentation. Specifically what you are working around which caused you to add this alternate representation of the ID. It might not belong in the documentation tag though, in which case you should add it as a comment.